db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql

```diff
@@ -94,8 +94,6 @@ CREATE TABLE dbo.nrt_investigation (
     cd                                  varchar(50)                                     NULL,
     cd_desc_txt                         varchar(100)                                    NULL,
     prog_area_cd                        varchar(20)                                     NULL,
-    jurisdiction_code                   varchar(20)                                     NULL,
-    jurisdiction_code_desc_txt          varchar(255)                                    NULL,
```

Comment:
What about corresponded changes in the `InvestigationReporting.java`?

Reply:
checking on it, when I remove it , the service is failing.

Reply:
it is working now, without deleteing the columns.